src/gtc/passes/oir_optimizations/utils.py

```diff
@@ -129,6 +141,10 @@ def write_fields(self) -> Set[str]:
             """Get a set of all written fields' names."""
             return {acc.field for acc in self._ordered_accesses if acc.is_write}
 
+        def mask_writes(self) -> Set[str]:
+            """Get a set of all fields' names written in mask statements."""
+            return {acc.field for acc in self._ordered_accesses if acc.is_write and acc.in_mask}
```

Comment:
I can't tell if this is used. If it's not used we can wind back all the changes with `in_mask`.

Reply:
It's an artefact from the horizontal regions PR that is yet to come I think. I removed it from this one.